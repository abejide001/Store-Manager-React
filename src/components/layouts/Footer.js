import React from 'react';
import '../../assets/css/style.css';

export default function Footer() {
  return (
    <div>
      <footer>
        <div className="container">
          <h3>
            Store Manager
            {' '}
            {new Date().getFullYear()}
          </h3>
        </div>
      </footer>
    </div>
  );
}
