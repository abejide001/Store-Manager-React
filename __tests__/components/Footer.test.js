/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from '../../src/components/layouts/Footer';

describe('<Footer />', () => {
  describe('render()', () => {
    test('renders the component', () => {
      const wrapper = shallow(<Footer />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
