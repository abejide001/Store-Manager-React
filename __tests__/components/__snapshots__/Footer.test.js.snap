// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> render() renders the component 1`] = `
<div>
  <footer>
    <div
      className="container"
    >
      <h3>
        Store Manager
         
        2019
      </h3>
    </div>
  </footer>
</div>
`;
